### ISSUE TITLE
terms and condition, FAQ.

### ISSUE THREAD

**Issue by damilarekila** [zuri-training/Excel_comp_Team108_DataSYNC#5]
Terms and conditions for using our solution, liability and risks of loss of data if what happens and how.

FAQ, frequently asked questions. at least five questions and explicitly explained answer to questions relating to usage of our solution. 

Deliverables, Quality UI designs, wireframes, lo fi, and hi fi designs.

**Comment by SmilingJudg** [comment]
My name is Ukaibe Emelda Promise aka SmilingJudge. As a product designer in my team. I contributed to the creation of the Style guide ( Iconography to be precise as I was assigned) I also created our team (Logo)
 ( https://www.figma.com/file/Ie8A81XMRsHiodqFwaeTO8)
 and the arrangements and tiding of the figma work space before the first presentation. And after the first presentation, I was assigned to design 2 pages/Screen: "Terms & Conditions of usage of  DataSYNC website and the FAQs (Frequently asked Questions)" to enable users of DataSYNC acquiant themselves with the do and don't of DataSYNC usage as well as get answers to questions they may likely wish to know as they log into the website for easy navigation.
So I have been able to put up the wireframe, Lo-fi (https://www.figma.com/file/Ie8A81XMRsHiodqFwaeTO8)
and Hi-fidelity of these two pages.
 ( https://www.figma.com/file/Ie8A81XMRsHiodqFwaeTO8) With a deadline and I have delivered. 
Waiting for any other assigned task after the final presentation for finishing touches.
